test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -67,6 +67,8 @@ public async Task GivenAValidBundle_WhenSubmittingABatch_ThenSuccessIsReturnedFo
             ValidateOperationOutcome(resource.Entry[9].Response.Status, resource.Entry[9].Response.Outcome as OperationOutcome, _statusCodeMap[HttpStatusCode.NotFound], "Resource type 'Patient' with id '12334' couldn't be found.", IssueType.NotFound);
         }
 
+        // ToDo: Refactor this test as it contains a bundle with duplicated resources.
+        /*
```

Comment:
Can you create a ticket for this? Just to do not forget it 😆

Reply:
This is part of the pending part of this work. I'll update the team with the decision we made on this.

Reply:
Added new parameter to FHIRClient and added this test back.